### DISCUSSION TITLE
训练一段时间便报错中断

### DISCUSSION THREAD

**Discussion by NAOLIU** [svc-develop-team/so-vits-svc#40] [Q&A]
![573a4acdb4e80846e9bb9851ef703bf4](https://user-images.githubusercontent.com/55797625/225925875-49295d75-3093-409c-9479-3adcb495f5d4.png)
![QQ截图20230317215429](https://user-images.githubusercontent.com/55797625/225925942-ec8b4c26-5587-48db-abd7-534625f702fa.jpg)

每次训练到一定程度便报错中断
使用的是https://www.bilibili.com/video/BV1Eb411f7gX的这个一键包

**Comment by Miuzarte** [comment]
一键包请询问原作者

**Comment by NAOLIU** [comment]
> 一键包请询问原作者

原作者没回复XD
不知道有无跟我一样情况的
